CHANGELOG.md

```diff
@@ -0,0 +1,55 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based roughly on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [0.13.0] - 2020-12-27
+### Added
+- Client-side interceptors (thanks @villainy!)
```

Comment:
@villainy Would you like me to add your real name here?

Reply:
Sure, I should probably get around to changing my git configs to "Michael Morgan" one of these days anyway. Thanks!